pulp_smash/tests/docker/api_v2/test_crud.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
Nitpick: can I get an encoding declaration, a change in format and unicode literals?

```python
# coding=utf-8
"""Test CRUD for Docker repositories.

This module contains tests for creating Docker repositories. It is intended to
also contain read, update, and delete tests.

"""
from __future__ import unicode_literals

# other imports here
```

(#24 is a related low-priority issue.)